src/dune_rules/merlin_ident.mli

```diff
@@ -6,7 +6,9 @@ type t
 
 val for_lib : Lib_name.t -> t
 
-val for_exes : names:string list -> t
+val for_exes : names:('a * string) list -> t
```

Comment:
Why was this signature changed?

Reply:
To avoid having to extract the names at the call points. Reverted in https://github.com/ocaml/dune/pull/6454/commits/d1cdb7707f2be98e2ed02280a3414c64fc7be099.